addon/model/model-range.ts

```diff
@@ -433,9 +461,57 @@ export default class ModelRange {
   }
 
   sameAs(other: ModelRange): boolean {
+    console.log('THISROOT', this.root);
+    console.log('OTHERROOT', other.root);
     return this.start.sameAs(other.start) && this.end.sameAs(other.end);
   }
 
+  compare(other: ModelRange): RangeComparison {
+    const startCompStart = this.start.compare(other.start);
```

Comment:
It took me a while to understand waht startCompStart meant, didn't immediately find a better name though

Reply:
yeah the non-abbreviated version is really quite verbose

Reply:
maybe startStartDiff?